Modules/_ctypes/_ctypes.c

```diff
@@ -5748,7 +5763,7 @@ comerror_dealloc(PyObject *self)
 }
 
 static PyType_Slot comerror_slots[] = {
-    {Py_tp_doc, (void *)PyDoc_STR(comerror_doc)},
+    {Py_tp_doc, PyDoc_STR(comerror_doc)},
```

Comment:
Keep the cast here, for the `const` warning?

Reply:
Right. Actually, it wasn't an issue with my gcc configuration and I forgot to revert that.